docs/APIs.md

```diff
@@ -1,80 +1,56 @@
 # Constructors
 
-## [vegaTooltip.vega](#vgtooltip)
+## [vegaTooltip](#tooltip)
 
-`vegaTooltip.vega(vgView[, options])`
+`vegaTooltip(view [, options])`
 
 | Parameter       | Type           | Description     |
 | :-------------- |:--------------:| :-------------- |
-| `vgView`        | [Vega View](https://github.com/vega/vega/wiki/Runtime#view-component-api) | The visualization view. |
+| `view`          | [Vega View](https://github.com/vega/vega/wiki/Runtime#view-component-api) | The visualization view. |
 | `options`       | Object         | Options to customize the tooltip. See [options](#options) for details. |
 
-__Returns:__ [`vegaTooltip`](#tooltip) object
-
-## [vegaTooltip.vegaLite](#vltooltip)
-
-`vegaTooltip.vegaLite(vgView, vlSpec[, options])`
-
-| Parameter       | Type           | Description     |
-| :-------------- |:--------------:| :-------------- |
-| `vgView`        | [Vega View](https://github.com/vega/vega/wiki/Runtime#view-component-api) | The visualization view. |
-| `vlSpec`        | Object         | The Vega-Lite specification of the visualization. |
-| `options`       | Object         | Options to customize the tooltip. See [options](#options) for details. |
-
-__Returns:__ [`vegaTooltip`](#tooltip) object
+__Returns:__ [`return`](#return) object
 
 ## [options](#options)
 
-`options` can customize the content and look of the tooltip. Here is a template of `options`. All of its properties are optional.
+`options` can customize the tooltip. Here is a template of `options`. All of its properties are optional.
 
 ```js
 var options =
 {
-  showAllFields: true | false,
-  fields: [
-    {
-      field: "field1",
-      title: "Field One",
-      formatType: "time" | "number" | "string",
-      format: string-specifier,
-      aggregate: operation,
-    },
-    ...
-  ],
-  delay: 250,
-  onAppear: function(event, item) { /* callback */ },
-  onMove: function(event, item) { /* callback */ },
-  onDisappear: function(event, item) { /* callback */ },
-  colorTheme: "light" | "dark"
+  /**
```

Comment:
I think this is a bit verbose - you duplicate documentation both here and in the table below.

Reply:
I will say that these are the defaults.